internal/atlas/atlas.go

```diff
@@ -33,6 +33,13 @@ type (
 		URL             string
 		RevisionsSchema string
 	}
+	// LintParams are the parameters for the `migrate lint` command.
+	LintParams struct {
+		DevURL  string
+		DirURL  string
+		Latest  uint
+		GitBase bool
```

Comment:
GitBase is not a real option, no?
```suggestion
		Latest  uint
```

Reply:
Yes, we don't use it yet